jenkins.sh

```diff
@@ -12,6 +12,9 @@ set -x
 # install the other dependencies
 npm install
 
+# we may be using a dev branch of js-sdk in which case we need to build it
+(cd node_modules/matrix-js-sdk && npm run build)
```

Comment:
It's `npm run compile` as `build` is used for [distribution](https://github.com/matrix-org/matrix-js-sdk/blob/master/package.json#L10).